apollos-church-api/src/data/Pass.js

```diff
@@ -0,0 +1,15 @@
+import {
+  dataSource,
+  schema,
+  resolver as baseResolver,
+} from '@apollosproject/data-connector-passes';
+
+const resolver = {
+  ...baseResolver,
+  Pass: {
+    ...baseResolver.Pass,
+    passkitFileUrl: () => null,
```

Comment:
why are we customizing this to a function that return `null`?

Reply:
Pass generation (adding to wallet) is currently broken in core. By customizing this a function that returns null, the button to "add to wallet" is removed. This is also probably be the temp solution for NewSpring.

Reply:
https://github.com/NewSpring/crete/pull/164/files